src/responder/common/negcache.c

```diff
@@ -900,7 +887,9 @@ errno_t sss_ncache_prepopulate(struct sss_nc_ctx *ncache,
                 continue;
             }
         } else {
-            for (dom = domain_list; dom; dom = get_next_domain(dom, 0)) {
+            for (dom = domain_list;
+                 dom != NULL;
+                 dom = get_next_domain(dom, SSS_GND_DESCEND)) {
```

Comment:
Hmm. I didn't add the inactive domains just because I was not sure whether it was a good idea or not. By you review, it seems to be.

Fixed locally!